### PR TITLE
[Erc20] add about 2900 mainnet tokens

### PR DESCRIPTION
**Pull request by bh2smith**
Here is all traded tokens on EVM that have appeared in at least 250 trades (since January 1, 2023). Only took about 5 minutes to fetch.

This uses the "new and improved" extended missing tokens query: see here https://github.com/cowprotocol/data-misc/pull/31 

cc @jeff-dude  and @gentrexha

### CODE REVIEW

**Comment by bh2smith** [comment]
Note that the failing CI is on an untouched file.

**Comment by jeff-dude** [comment]
> Note that the failing CI is on an untouched file.

apologies, fixing this in #3307

File: models/tokens/ethereum/tokens_ethereum_erc20.sql [hunk 1]
```diff
@@ -21558,4 +21558,2969 @@ FROM (VALUES
         ,('0xfb272d3acfcbf3e0f92bd01ffbcde92575c371f9', 'JPOW', 9)
         ,('0xfca99bf40253563f619b5fbaed621021b2987a09', 'zjz', 18)
         ,('0xfe60fba03048effb4acf3f0088ec2f53d779d3bb', '3d3d', 18)
+                ,('0x235c8ee913d93c68d2902a8e0b5a643755705726', 'BAG', 18)
```
Comment (jeff-dude) [inline models/tokens/ethereum/tokens_ethereum_erc20.sql:21561]: v nitpicky, but can we fix the spacing here to align? maybe it'd happen each time the script ran?

Comment (bh2smith) [reply models/tokens/ethereum/tokens_ethereum_erc20.sql:21561]: fixed.

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**